### ISSUE TITLE
Compilation fails on kernel 5.6.7

### ISSUE THREAD

**Issue by chrissprague** [datto/dattobd#221]
When compiling, I receive the following error:

```
dattobd/src/dattobd.c: In function 'agent_init':
dattobd/src/dattobd.c:5169:51: error: passing argument 4 of 'proc_create' from incompatible pointer type [-Werror=incompatible-pointer-types]
 5169 |  info_proc = proc_create(INFO_PROC_FILE, 0, NULL, &dattobd_proc_fops);
      |                                                   ^~~~~~~~~~~~~~~~~~
      |                                                   |
      |                                                   const struct file_operations *
In file included from dattobd/src/includes.h:16,
                 from dattobd/src/dattobd.c:7:
./include/linux/proc_fs.h:64:24: note: expected 'const struct proc_ops *' but argument is of type 'const struct file_operations *'
   64 | struct proc_dir_entry *proc_create(const char *name, umode_t mode, struct proc_dir_entry *parent, const struct proc_ops *proc_ops);
      |                        ^~~~~~~~~~~
```

My system kernel is `5.6.7-200.fc31.x86_64`

I have been compiling from source on this machine for a while. It broke after I restarted recently, presumably due to a kernel upgrade. The previous two kernels I have on my system were `5.5.11` and `5.6.6`. I am guessing this broke since 5.5.11, as the `proc_ops` structure has changed since then ([5.6.7](https://elixir.bootlin.com/linux/v5.6.7/source/include/linux/proc_fs.h#L15) vs [5.5.11](https://elixir.bootlin.com/linux/v5.5.11/source/drivers/usb/gadget/udc/lpc32xx_udc.c#L532), the latter which I can't find a struct of the same name, but the linked struct looks similar to ours)

**Comment by yves84** [comment]
Same problem here.

Using Kernel 5.7.0 on Manjaro.

**Comment by yesrod** [comment]
Same here, using linux-oem-20.04 (5.6.0-1018-oem) on Ubuntu 20.04.

Build log from DKMS is attached. 
[dattobd-dkms.0.crash.txt](https://github.com/datto/dattobd/files/4892327/dattobd-dkms.0.crash.txt)

**Comment by dakotarwilliams** [comment]
Fixed #229
